### PR TITLE
Feature/browse documents

### PR DESCRIPTION
**Pull request by fervelgo**
## Coauthors
Jacob Shomali
Fernando Velasco

## Description

We added a default search value to the "getDsData" function such that we load a default set of documents when a user signs in.

## Video

https://www.loom.com/share/277e6ad1cb3340129824941786f523ff

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by HeyAnwar — APPROVED** [review]
Great work !!

**Review by collerhatguy — APPROVED** [review]
Its ready spaghetti

**Review by juliane1999 — APPROVED** [review]
Great work!

**Review by bevanger — APPROVED** [review]
Great job getting this feature added. The code looks great

**Review by rkshockey — CHANGES_REQUESTED** [review]
I checked with Paul about the .env.sample document - it should have the variable names, but they should be equal to example variables and not what we're actually using! The client id and uri's should not be up here.

File: .env.sample [hunk 1]
```diff
@@ -1,3 +1,4 @@
-REACT_APP_CLIENT_ID=example
-REACT_APP_OKTA_ISSUER_URI=https://example-882474.okta.com
-REACT_APP_API_URI=http://localhost:8005
\ No newline at end of file
+REACT_APP_CLIENT_ID=0oavso7p45Q1PRKSk4x6
```
Comment (rkshockey) [inline .env.sample:1]: Only the variable names here should be the real thing!

**Review by fervelgo — COMMENTED** [review]
Ready ready ready

**Review by rkshockey — APPROVED** [review]

**Review by BradfordMez — APPROVED** [review]
Great job y'all! Smart idea adding a default search! Looks good!

**Review by RimshaSaleem — APPROVED** [review]

**Pull request merged**